### DISCUSSION TITLE
Is cefsharp 120.1.80.0 or later versions compatible with Windows7?

### DISCUSSION THREAD

**Discussion by lofti198** [cefsharp/CefSharp#4775] [Q&A] [answered]
Hi! Is cefsharp 120.1.80.0 compatible with Windows7 (with .NET 4.8 framework installed)?

**Comment by amaitland** [comment, marked as answer]
No. Version `109` is the last version to run on `Windows 7`

See https://github.com/cefsharp/CefSharp/discussions/4336
